### ISSUE TITLE
Orgbook to validate organization

### ISSUE THREAD

**Issue by rgreensp** [bcgov/connectivity-intake#343]

**Comment by rgreensp** [comment]
Blocked on decision to include or not include satellite to the home in initial intake. (Starlink would not be in BC Registries.)

**Comment by eazeredo** [comment]
@rgreensp and @MarsRomer , can we address this validation soon? (cc @zoeyli-46 , @elliottSlee )

**Comment by MarsRomer** [comment]
@eazeredo correct me if I'm wrong, but I believe this is blocking future work - i.e. not this sprint, and prob not next either.  Def needs to be unblocked and I will be working on it - I'm meeting Zoey this afternoon to discuss.

**Comment by eazeredo** [comment]
You are correct - it's not for this or the next one. 
I don't know if we have mapped the complexity to use this service (access, permissions, etc). If it's simple, we don't need to worry.

**Comment by MarsRomer** [comment]
@zoeyli-46  I added 3 stickys since there were a few steps with the Orgbook which I am not sure if we will do. I think we should discuss the value/need for validating with Orgbook.  Most of the validation of the organization will be done as part of the applicant review process followed by back and forth negotiation.

**Comment by elliottSlee** [comment]
This makes sense @MarsRomer. The intention of organization validation was two-fold: (1) reduce the business area’s workload of ensuring orgs are legal entities and (2) ensure that reporting out of database is deduplicated of potential multiple entries from same company (whether due to different sub-companies or misspelling). It appears the business area will continue to handle (1) themselves so we don’t need to. And for (2) @wenzowski and @zoeyli-46 found the elegant solution of asking about parent org.
